extensions/extended-blocks/paid-blocks/edit.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import UpgradePlanBanner from './upgrade-plan-banner';
+import { isUpgradable } from '../../shared/plan-utils';
+
+export default OriginalBlockEdit => props => {
+	if ( ! isUpgradable( props?.name ) ) {
+		return <OriginalBlockEdit { ...props } />;
+	}
+
+	return (
+		<Fragment>
+			<InspectorControls>
+				<UpgradePlanBanner description={ null } blockName={ props.name } />
```

Comment:
Why do we need to mark explicitly as `null` here?

Reply:
Because the banner has defined a description as default. We need to set as `null` to remove it when it's rendered the block inspector.